nixos/modules/profiles/minimal.nix

```diff
@@ -18,6 +18,15 @@ with lib;
 
   documentation.nixos.enable = mkDefault false;
 
+  # Perl is a default package.
+  environment.defaultPackages = mkDefault [ ];
+
+  # The lessopen package pulls in Perl.
+  programs.less.lessopen = mkDefault null;
+
+  # This pulls in nixos-containers which depends on Perl.
+  boot.enableContainers = mkDefault false;
+
```

Comment:
Can we please add a release note entry for `boot.enableContainers`?

https://discourse.nixos.org/t/breaking-changes-announcement-for-unstable/17574 would have been fitting as well.

Reply:
Created #269970 for a simple eval warning when containers are used but disabled (e.g. by importing the minimal profile).